src/sequencer_alsa/sequencer_alsa.i

```diff
@@ -37,10 +37,14 @@ event_input(snd_seq_t *handle)
     int err;
     snd_seq_event_t *ev;
     err = snd_seq_event_input(handle, &ev);
+    if (err == -EAGAIN)
+    {
+        Py_INCREF(Py_None);
+        return Py_None;
```

Comment:
This produces a compilation warning:
```
src/sequencer_alsa/sequencer_alsa_wrap.c:3039:9: warning: return from incompatible pointer type [enabled by default]
         return Py_None;
         ^
```

And while it *seems to work fine on my machine*, TBH I'm not sure it's safe to pass Py_None through SWIG's SWIG_NewPointerObj() wrapper like this.